LedRGB.cpp

```diff
@@ -9,21 +9,15 @@ LedRGB::LedRGB(unsigned redPin, unsigned greenPin, unsigned bluePin)
 }
 
 void LedRGB::setRGB(unsigned r, unsigned g, unsigned b) {
-    r /= 2.55;
-    g /= 2.55;
-    b /= 2.55;
-    if (checkBrightness) {
-        RedLed.setBrightness(r);
-        GreenLed.setBrightness(g);
-        BlueLed.setBrightness(b);
-    }
+    setBrightness(r);
+    setBrightness(g);
+    setBrightness(b);
```

Comment:
Say I set the rbg to (255, 100, 25), does that mean the red has 100% brightness, green has Y% brightness, and blue has Z% brightness. Or would there be one set brightness for all the three colors?